google/google-how-to/gce/index.rst

```diff
@@ -11,4 +11,5 @@ GCE instances
    enable-pro-features
    build-ubuntu-pro-golden-image
    create-customised-docker-container
-   upgrade-from-focal-to-jammy
\ No newline at end of file
+   upgrade-from-focal-to-jammy
+   gce-image-retention-policy
```

Comment:
Since this is an explanation and not a how-to guide, remove this entry from here and add it to the index.rst in the google-explanation folder